### PR TITLE
feat(idl): update idl highlight

### PR DESCRIPTION
**Pull request by cathaysia**
fix breaking changes

### CODE REVIEW

File: queries/idl/highlights.scm [hunk 1]
```diff
@@ -49,6 +49,28 @@
...
+  "SEQUENTIAL"
+  "HASH"
+  "FINAL"
+  "APPENDABLE"
+  "MUTABLE"
+  "c"
+  "c++"
+  "java"
+  "idl"
+  "*"
```
Comment (lucario387) [inline queries/idl/highlights.scm:62]: duplicated

File: queries/idl/highlights.scm [hunk 2]
```diff
@@ -132,11 +154,41 @@
...
+    "min"? @attribute.builtin
+    "max"? @attribute.builtin
+    "unit"? @attribute.builtin
+    "bit_bound"? @attribute.builtin
+    "external"? @attribute.builtin
+    "nested"? @attribute.builtin
+    "verbatim"? @attribute.builtin
+    "service"? @attribute.builtin
+    "oneway"? @attribute.builtin
+    "ami"? @attribute.builtin))
```
Comment (lucario387) [inline queries/idl/highlights.scm:185]:
```suggestion
    [
      "id"
      "autoid"
      "optional"
      "position"
      "value"
      "extensibility"
      "final"
      "appendable"
      "mutable"
      "key"
      "must_understand"
      "default_literal"
      "default"
      "range"
      "min"
      "max"
      "unit"
      "bit_bound"
      "external"
      "nested"
      "verbatim"
      "service"
      "oneway"
      "ami"
    ] @attribute.builtin))
```

**Comment by clason** [comment]
Indent queries need to be adapted, too.

File: queries/idl/highlights.scm (hunk 1 above)
Comment (cathaysia) [reply queries/idl/highlights.scm:62]: Can you be more detailed?

Comment (lucario387) [reply queries/idl/highlights.scm:62]: There is another `"*"` on line 71, duplicated to this one.

Comment (clason) [reply queries/idl/highlights.scm:62]: The `"*"` node appears at the end of the list again.

**Review by lucario387 — APPROVED** [review]

**Pull request merged**